scalastyle-config.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<scalastyle commentFilter="enabled">
+    <name>Scalastyle ARM standard configuration</name>
```

Comment:
I'm super-against `scalastyle`. Honestly you'll get nothing from abuse from me about everything it suggests, it's dead and the author isn't even active anymore https://github.com/scalastyle/scalastyle/graphs/contributors